Tests/SystemInfo/Collector/ConfigurationSymfonyKernelSystemInfoCollectorTest.php

```diff
@@ -10,7 +10,7 @@
 
 use EzSystems\EzSupportToolsBundle\SystemInfo\Collector\ConfigurationSymfonyKernelSystemInfoCollector;
 use EzSystems\EzSupportToolsBundle\SystemInfo\Value\SymfonyKernelSystemInfo;
-use PHPUnit_Framework_TestCase;
+use PHPUnit\Framework\TestCase as PHPUnit_Framework_TestCase;
```

Comment:
Wouldn't it be simpler to change `extends PHPUnit_Framework_TestCase` into `extends TestCase`? Just one line more ;)

Reply:
sure nitty :)

Reply:
Fixed in e63eadc
Also bumped php-cs-fixer in 8438f57 so runs on PHP 7.3 works